doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2409,3 +2409,55 @@ setting indicates embargoes are not supported. A value of -1 allows embargoes of
 can enter for an embargo end date. This limit will be enforced in the popup dialog in which users enter the embargo date. For example, to set a two year maximum:
 
 ``curl -X PUT -d 24 http://localhost:8080/api/admin/settings/:MaxEmbargoDurationInMonths``
+
+:DataverseMetadataValidatorScript
++++++++++++++++++++++++++++++++++
+
+An optional external script that validates Dataverse collection metadata as it's being updated or published. The script provided should be an executable that takes a single command line argument, the name of the file containing the metadata exported in the native json format. The script should exit with a non-zero error code if the validation fails. If that happens, a failure message (customizable in the next two settings below, `:DataverseMetadataValidationFailureMsg` and `DataverseMetadataUpdateValidationFailureMsg`) will be shown to the user.
```

Comment:
```suggestion
An optional external script that validates Dataverse collection metadata as it's being updated or published. The script provided should be an executable that takes a single command line argument, the full path to the Dataverse collection validation script. The script should exit with a non-zero error code if the validation fails. If that happens, a failure message (customizable in the next two settings below, `:DataverseMetadataValidationFailureMsg` and `:DataverseMetadataUpdateValidationFailureMsg`) will be shown to the user.
```

Reply:
No, that would be completely incorrect - sorry.

Reply:
(fixed the missing colon though)

Reply:
Ah, I thought I had found a copy/paste error. I get it now. There's a lot packed into a single sentence. It's fine to leave it alone but we could also consider separated describing:

- what the setting points to (the script)
- how the script operates (by taking a JSON file as an argument)